.github/workflows/release.yml

```diff
@@ -17,13 +17,35 @@ jobs:
       - id: release
         uses: google-github-actions/release-please-action@v4
 
+
       # If release-please created a release, publish to npm
-      - if: ${{ steps.release.outputs.release_created }}
+      - if: steps.release.outputs.release_created
         uses: actions/checkout@v4
-      - if: ${{ steps.release.outputs.release_created }}
+      - if: steps.release.outputs.release_created
         name: actions/setup
         uses: ./.github/actions/setup
+      - if: steps.release.outputs.release_created
+        run: npm pack
+        shell: bash
+      - if: steps.release.outputs.release_created
+        name: Get release version and release package file name
+        run: |
+          PACKAGE_VERSION=$(jq '.version' package.json | tr -d '"')
+          echo "PACKAGE_VERSION=${PACKAGE_VERSION}" >> "$GITHUB_ENV"
+          echo "PACKAGE_FILE=bson-${PACKAGE_VERSION}.tgz" >> "$GITHUB_ENV"
+      - if: steps.release.outputs.release_created
+        name: Create detached signature
+        uses: mongodb-labs/drivers-github-tools/garasign/gpg-sign@main
```

Comment:
Will this be versioned? I think there is a v1 already

Reply:
Yep, just added it. Seems like v1 was published just yesterday